src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -242,6 +243,11 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('kernel.default_locale', $config['default_locale']);
         $container->setParameter('kernel.error_controller', $config['error_controller']);
 
+        if (isset($config['trusted_proxies'], $config['trusted_headers'])) {
+            $container->setParameter('kernel.trusted_proxies', $config['trusted_proxies']);
```

Comment:
Can we explode here instead of at runtime? We might also want to trim?

Reply:
we can't really if we want to accept env vars here

Reply:
`trim()` added, at runtime: I tried, but we don't support csv-parsed env vars in array-node definitions. I'm sure that could be done, but that'd require a lot of work on the Config component before. For now at least, we need to support string+explode.